src/webui/components/creator/elementary/EditButton.tsx

```diff
@@ -0,0 +1,20 @@
+
+import { Button } from '^/webui/designsystem/module';
+
+export type Props = {
+    editHandler: () => void;
+};
+
+export default function Component({ editHandler }: Props)
+{
+    const handleClick = async () =>  
```

Comment:
What's the use of this wrapper function? I think we can use the `editHandler` function as `clickHander` directly?